File: coffee-system/src/main/java/com/steelzack/coffee/system/objects/EmployeeLayer.java
package com.steelzack.coffee.system.objects;

import com.steelzack.coffee.system.input.CoffeeMachines.CoffeMachine.Coffees.Coffee;
import com.steelzack.coffee.system.input.CoffeeMachines.CoffeMachine.PaymentTypes.Payment;
import com.steelzack.coffee.system.input.Employees;
import lombok.Builder;
import lombok.Getter;

/**
 * Created by joaofilipesabinoesperancinha on 05-05-16.
 */
@Builder
@Getter
public class EmployeeLayer {
    private Employees.Employee employee;

    private Coffee coffee;

    private Payment payment;
}

File: jeorg-kotlin-apps/coffee-paradigms/coffee-system/src/main/kotlin/org/jesperancinha/coffee/system/objects/EmployeeLayer.kt
package org.jesperancinha.coffee.system.objects

import org.jesperancinha.coffee.system.input.CoffeeMachines.CoffeeMachine.Coffees.Coffee
import org.jesperancinha.coffee.system.input.CoffeeMachines.CoffeeMachine.PaymentTypes.Payment
import org.jesperancinha.coffee.system.input.Employees.Employee

/**
 * Created by joaofilipesabinoesperancinha on 05-05-16.
 */
data class EmployeeLayer(
    val employee: Employee,
    val coffee: Coffee,
    val payment: Payment,
)